src/components/Collections/ListItems/PlioListItem.vue

```diff
@@ -262,7 +170,9 @@ export default {
     },
     updatedAt() {
       // human readable date string
-      return new Date(this.plioDetails.updatedAt).toDateString();
+      // removed the first four characters of the output as they represent the day
+      // and lead to inconsistent length of the string
```

Comment:
best to not put "removed the first four..." thing in the comments. We know the old code so we can identify what has changed, but for someone reading in the future might not know that. Just add the date format as a comment